app/model/exchange/Assessor.scala

```diff
@@ -24,15 +25,50 @@ case class Assessor(userId: String, skills: List[String], civilServant: Boolean)
 object Assessor {
   implicit val assessorFormat = Json.format[Assessor]
 
-  def apply(assessor: model.persisted.Assessor): Assessor = Assessor(assessor.userId, assessor.skills, assessor.civilServant)
+  def apply(assessor: model.persisted.assessor.Assessor): Assessor = Assessor(assessor.userId, assessor.skills, assessor.civilServant)
 }
 
-
-case class AssessorAvailability(userId: String, availability: Map[String, List[LocalDate]])
+case class AssessorAvailability(location: String, date: LocalDate)
 
 object AssessorAvailability {
   implicit val assessorAvailabilityFormat = Json.format[AssessorAvailability]
 
-  def apply(assessor: model.persisted.Assessor): AssessorAvailability = AssessorAvailability(assessor.userId, assessor.availability)
+  def apply(persisted: model.persisted.assessor.AssessorAvailability): AssessorAvailability = {
+    AssessorAvailability(persisted.location, persisted.date)
+  }
+
+  def apply(request: AssessorAvailabilityRequest): AssessorAvailability = {
+    AssessorAvailability(request.location, request.date)
+  }
+}
+
+case class AssessorAvailabilities(userId: String, availability: List[AssessorAvailability])
+
+object AssessorAvailabilities {
+  implicit val assessorAvailabilityFormat = Json.format[AssessorAvailabilities]
+
+  def apply(assessor: model.persisted.assessor.Assessor): AssessorAvailabilities =
+    AssessorAvailabilities(assessor.userId, assessor.availability.map(a => AssessorAvailability.apply(a)))
+}
+
+
+
+case class AssessorOld(userId: String, skills: List[String], civilServant: Boolean)
```

Comment:
Again i don't think we should have any 'old' classes, unless i'm missing something